index.html

```diff
@@ -598,60 +595,53 @@
-            >Оплата і повернення коштів
-          </a>
-          <a class="footer__navigation__link" href="#">
-            Політика конфіденсійності
-          </a>
-          <a class="footer__navigation__link" href="#">Сервіс користувача</a>
-        </nav>
-        <nav class="footer__navigation">
-          <a class="footer__navigation__link" href="#">Про компанію </a>
-          <a class="footer__navigation__link" href="#">Магазин</a>
-          <a class="footer__navigation__link" href="#">Співпраця </a>
-          <a class="footer__navigation__link" href="#">Контакти </a>
-          <a class="footer__navigation__link" href="#">Кошик</a>
-        </nav>
+          <nav class="footer__navigation">
+            <a class="footer__navigation__link" href="#">Програма лояльності</a>
+            <a class="footer__navigation__link" href="#">Діючі акції </a>
+            <a class="footer__navigation__link" href="#">Новини </a>
+            <a class="footer__navigation__link" href="#">Часті запитання </a>
+            <a class="footer__navigation__link" href="#">Карта сайту</a>
+          </nav>
+          <nav class="footer__navigation footer__navigation-second">
+            <a class="footer__navigation__link" href="#">Про компанію</a>
+
+            <a class="footer__navigation__link" href="#">Доставка</a>
+            <a class="footer__navigation__link" href="#"
+              >Оплата і повернення коштів
+            </a>
+            <a class="footer__navigation__link" href="#">
+              Політика конфіденсійності
+            </a>
+            <a class="footer__navigation__link" href="#">Сервіс користувача</a>
+          </nav>
+          <nav class="footer__navigation">
+            <a class="footer__navigation__link" href="#">Про компанію </a>
+            <a class="footer__navigation__link" href="#">Магазин</a>
+            <a class="footer__navigation__link" href="#">Співпраця </a>
+            <a class="footer__navigation__link" href="#">Контакти </a>
+            <a class="footer__navigation__link" href="#">Кошик</a>
+          </nav>
         </div>
         <div class="footer__quality-mark">
           <div class="footer__quality-mark__img-container">
-          <img
-            class="footer__quality-mark__img"
-            src="./img/footer-logo.png"
-            alt="Знак якості"
-          />
+            <img
+              class="footer__quality-mark__img"
+              src="./img/footer-logo.png"
+              alt="Знак якості"
+            />
           </div>
           <p class="footer__quality-mark__text">
             © 2013 Baby Brok. Всі права захищені
-            <br />
-            <br />
+          </p>
+          <p class="footer__quality-mark__text">
             Всі торгові марки є торговими марками їх відповідних власників
           </p>
         </div>
         <div class="footer__more-information">
           <div class="footer__social-networks">
```

Comment:
I believe having `<nav>` here is a bit better then `<div>`. What do you think?

Reply:
I think it's good idea, because we have some `a` tegs. Thanks!

Reply:
** tags